magento/pwa-studio
# [doc] Refactor site content navigation

## Description

This PR refactors the doc site navigation lists to improve content discovery and separate the different types of docs (i.e. conceptual, tutorials, and reference).

## Related Issue

None

## Acceptance 

@awilcoxa

### Verification Stakeholders

@awilcoxa

### Verification Steps

1. Navigate to the docs directory: `cd pwa-devdocs`
2. Remove any existing `_site` directory: `rm -fr _site`
3. Build the HTML preview site: `npm run develop`
4. Verify new top navigation
5. Verify **Getting Started** and **Reference API** nav items have a dropdown to more specific items
6. Click through the various items on the top nav
7. Verify items have new left navigation contents and the entry scrolls into view

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
